### PR TITLE
fix: populate media in document service

### PR DESCRIPTION
**Pull request by jhoward1994**
### What does it do?

Change Media population logic in the document service so webhooks receive full media info

### Related issue(s)/PR(s)

Fixes https://github.com/strapi/strapi/issues/21546

### CODE REVIEW

**Comment by jhoward1994** [comment]
Alternative idea

```
selectAllMedia?: boolean;
...
acc[attributeName] = { select: opts.selectAllMedia ? ['*'] : ['id'] };
```

**Review by Marc-Roig — DISMISSED** [review]
lets go with this for now as it looks simpler :) ! nice catch

File: packages/core/core/src/services/document-service/utils/populate.ts [hunk 1]
```diff
@@ -36,7 +36,7 @@ export const getDeepPopulate = (uid: UID.Schema, opts: Options = {}) => {
       }
 
       case 'media': {
-        acc[attributeName] = { select: ['id'] };
+        acc[attributeName] = { select: ['*'] };
```
Comment (Marc-Roig) [inline packages/core/core/src/services/document-service/utils/populate.ts:39]: could you add a note thats important to populate all data so the webhooks contain that informatoin? maybe linking the issue

**Comment by jhoward1994** [comment]
Thanks, will do!

**Review by innerdvations — DISMISSED** [review]
LGTM!

**Review by innerdvations — APPROVED** [review]

**Pull request merged**